release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3864,4 +3864,28 @@ $ oc adm release info 4.10.53 --pullspecs
 [id="ocp-4-10-53-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] for instructions.
```

Comment:
```suggestion
To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI].
```

Reply:
This would have to be a CI item for RNs because this is stock text that is used across all branches.